src/main/java/org/mule/extension/sftp/internal/command/SftpWriteCommand.java

```diff
@@ -82,7 +72,7 @@ public void write(String filePath, InputStream content, FileWriteMode mode,
       if (e instanceof DeletedFileWhileReadException) {
         throw new ModuleException(e.getCause().getMessage(), FileError.FILE_DOESNT_EXIST, e.getCause());
```

Comment:
We should throw a custom exception here instead of a module exception.  This exception type is defined as CONNECTIVITY, it shouldn't be defined like this. @acurci-at-mulesoft